### PR TITLE
Individual Update 31/05/2023

### PR DESCRIPTION
**Pull request by balarammahato**
Merging feature_pullrequest_01062023 branch.

### CODE REVIEW

**Comment by LUJ13** [comment]
Well done

**Comment by pubba1998** [comment]
okey

**Comment by ClydeLiu1** [comment]
love it

File: BalaramUpdate.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          Help:
+          N/A
+
+01/06/2023  Meeting update
+ 
+ Balaram: I helped clyde to created feature branch based on released_01062023.
+          I have created the document for how to create pull request, review and complete pull request.
+
+          Help:
+          N/A
```
Comment (ClydeLiu1) [inline BalaramUpdate.txt:47]: ok test

File: BalaramUpdate.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+26/05/2023  Meeting update
+ 
+ Balaram: I pull the request from master.
+          I mergeed my conflict while making pull request from master.
+          I pushed the sarcastic chatbot to my branch.
+          I explored some features of actions in github.
+
```
Comment (LUJ13) [inline BalaramUpdate.txt:27]: Please create new line

File: BalaramUpdate.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          I have updated action list and achivement.
+
+          Help:
+          N/A
+
+01/06/2023  Meeting update
+ 
+ Balaram: I helped clyde to created feature branch based on released_01062023.
+          I have created the document for how to create pull request, review and complete pull request.
+
```
Comment (ClydeLiu1) [inline BalaramUpdate.txt:45]: ok test

File: BalaramUpdate.txt (hunk 1 above)
Comment (balarammahato) [reply BalaramUpdate.txt:47]: Thank you

**Pull request merged**